ChangeLog.d/bugfix_type_limit_warning_ssl_context_info.txt

```diff
@@ -0,0 +1,3 @@
+Bugfix
+   * read_next_b64_code() isn't correctly checking against EOF because it checks against 'char' rather than 'int'. This is identified via type-limit warning reported by gcc.
```

Comment:
Please word changelog entries for users of the libraries, not for maintainers. `read_next_b64_code` is not meaningful to users, `ssl_context_info` is. This is a bug in `ssl_context_info` which, on some platforms, is silently truncating base64 data at character `\377` rather than ignoring it.